code-challenges.test.js

```diff
@@ -36,14 +137,50 @@ const nums2 = [109, 5, 9, 67, 8, 24]
 
 // b) Create the function that makes the test pass.
 
+const numsOrder = (array) => {
+    return array.filter(value => Math.min, Math.max)
+}
```

Comment:
I like the idea of using Math.min and Math.max or filter.  Both are great approaches but are two separate directions that can be taken.